plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/InviteCommand.java

```diff
@@ -42,14 +53,52 @@ public void execute(@CanManage User deputy, Guild guild, String[] args) {
         when(args.length < 1, this.messages.generalNoNickGiven);
         when(guild.getMembers().size() >= this.config.maxMembersInGuild, formatter.format(this.messages.inviteAmount));
 
-        User invitedUser = UserValidation.requireUserByName(args[0]);
+        if (args[0].equals(this.config.inviteCommandAllArgument)) {
+
+            Double range;
```

Comment:
Zbędne całkowicie jest tutaj użycie nieprymitywnej formy, wystarczyło ustawić to na wartość z configu a catch pozostawić pusty. Warto jednak zerknąć na komentarz przy potworku `Double finalRange = range` w celu lepszej możliwości usprawnienia.